CHANGELOG.md

```diff
@@ -74,6 +76,10 @@ You can limit bitrates on outgoing tracks using [Localparticipant.setParameters]
 
 - Fixed an issue where loading twilio-video.js in firefox with `media.peerconnection.enabled` set to false in `about:config` caused page errors. (JSDK-2591)
 
+Known Issues
```

Comment:
@makarandp0 Please move this to COMMON_ISSUES.md and describe the exact scenario.

Reply:
@manjeshbhargav - I have moved this to COMMON_ISSUES.md and also added a common issue that we see on ios 13.0.1